<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1">
    <item>
     <widget class="QWidget" name="other_widget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16666666</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="5,2">
       <item>
        <widget class="QWidget" name="clock_widget" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="button_widget" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>300</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="ReturnButton">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>返回上一页</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="StartButton">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>开始</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="RestartButton">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>重新开始</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ExitButton">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="board_widget" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="player_widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1019</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>模式</string>
    </property>
    <widget class="QMenu" name="ha_menu">
     <property name="title">
      <string>人机对战</string>
     </property>
     <addaction name="h_black"/>
     <addaction name="h_white"/>
    </widget>
    <addaction name="ha_menu"/>
    <addaction name="hh_action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="hh_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>人人对战</string>
   </property>
  </action>
  <action name="h_black">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>玩家执黑</string>
   </property>
  </action>
  <action name="h_white">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>玩家执白</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
